src/Smearing.jl

```diff
@@ -130,10 +130,12 @@ function MethfesselPaxton(order::Integer)
     end
 end
 
-# TODO: Marzari-Vanderbilt "cold smearing"
+struct Cold <: SmearingFunction end
```

Comment:
Hm, ColdSmearing maybe? Or MarzariVanderbilt? Cold is a bit short...

Reply:
I think MarzariVanderbilt is good. That's also used by QE for input.

Reply:
Also could you add the reference of the equations to the top of the page (see the other comments).

Reply:
I chose to name it cold because that's most often what Prof. Marzari uses in our lab.
I agree though that it's more commonly called Marzari-Vanderbilt, so I'll change it to that for now.